### PR TITLE
Feature sb 66 vendor public page

### PR DESCRIPTION
**Pull request by ValentinaTenus**
### **General**
- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet
### Frontend
- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

1. changed styles on public vendor profile page
2. 
image
image

### CODE REVIEW

File: src/pages/product/components/vendor-preview-mode-modal/styles.ts [hunk 1]
```diff
@@ -1,21 +1,62 @@
...
+	"& .MuiTouchRipple-root": {
+		display: "none",
+	},
+	"&:active": {
+		background: "transparent",
+		backgroundColor: "transparent",
+	},
+	"&:hover, &:focus": {
+		background: "transparent",
+		backgroundColor: "transparent",
```
Comment (VladManukian) [inline src/pages/product/components/vendor-preview-mode-modal/styles.ts]:
```suggestion
		backgroundColor: theme.palette.transparent,
```

File: src/pages/product/components/vendor-preview-mode-modal/styles.ts [hunk 2]
```diff
@@ -1,21 +1,62 @@
...
+	flexDirection: "column",
 }));
 
 const StyledCrossIconButton = styled(IconButton)(() => ({
+	"& .MuiTouchRipple-root": {
+		display: "none",
+	},
+	"&:active": {
+		background: "transparent",
+		backgroundColor: "transparent",
```
Comment (VladManukian) [inline src/pages/product/components/vendor-preview-mode-modal/styles.ts]:
```suggestion
		backgroundColor: theme.palette.transparent,
```

**Review by VladManukian — COMMENTED** [review]
As i said in this [PR](https://github.com/ZenBit-Tech/sunny-bunnies_fe/pull/38), In my opinion we need use theme now for font size, weight and colors.

File: src/helpers/sort-sizes-by-asc-order.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const sortOrder = ["XS", "S", "M", "L", "XL"];
+
+function sortSizesByAscOrder<T extends number | string>(items: T[]): T[] {
+	return items.slice().sort((a, b) => {
+		return sortOrder.indexOf(a.toString()) - sortOrder.indexOf(b.toString());
+	});
+}
+
+export { sortSizesByAscOrder };
```
Comment (lyudad) [inline src/helpers/sort-sizes-by-asc-order.ts:12]: I saw the same changes in PR 38, you doest create branched from develop. its very very bad

**Comment by VladManukian** [comment]
theme.palette instead of colors for style.ts an sx props

**Review by lyudad — APPROVED** [review]

**Pull request merged**